[2022-03-30 16:47:58,926] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.pre.pre_get_data scheduled__2022-03-29T16:04:54.591828+00:00 [queued]>
[2022-03-30 16:47:58,943] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.pre.pre_get_data scheduled__2022-03-29T16:04:54.591828+00:00 [queued]>
[2022-03-30 16:47:58,944] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-30 16:47:58,945] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-30 16:47:58,946] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-30 16:47:58,964] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): pre.pre_get_data> on 2022-03-29 16:04:54.591828+00:00
[2022-03-30 16:47:58,971] {standard_task_runner.py:52} INFO - Started process 390 to run task
[2022-03-30 16:47:58,976] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 'pre.pre_get_data', 'scheduled__2022-03-29T16:04:54.591828+00:00', '--job-id', '1087', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmprfkir58_', '--error-file', '/tmp/tmp_tyqtl49']
[2022-03-30 16:47:58,978] {standard_task_runner.py:77} INFO - Job 1087: Subtask pre.pre_get_data
[2022-03-30 16:47:59,082] {logging_mixin.py:109} INFO - Running <TaskInstance: tutorial.pre.pre_get_data scheduled__2022-03-29T16:04:54.591828+00:00 [running]> on host ff48034dfdbf
[2022-03-30 16:47:59,329] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=pre.pre_get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-03-29T16:04:54.591828+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-29T16:04:54.591828+00:00
[2022-04-23 14:16:31,520] {taskinstance.py:1026} INFO - Dependencies not met for <TaskInstance: tutorial.pre.pre_get_data scheduled__2022-03-29T16:04:54.591828+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2022-04-23 14:16:31,523] {taskinstance.py:1026} INFO - Dependencies not met for <TaskInstance: tutorial.pre.pre_get_data scheduled__2022-03-29T16:04:54.591828+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2022-04-23 14:16:31,526] {local_task_job.py:99} INFO - Task is not able to be run
